Packs/JamfProtect/Integrations/JamfProtectEventCollector/JamfProtectEventCollector.py

```diff
@@ -420,147 +419,66 @@ def parse_response(response: dict) -> tuple:
-                                     for event in events))).strftime(DATE_FORMAT) if events else current_date
-
-    new_last_fetch_date = max(created, latest_event)
-    new_last_run = {"last_fetch": new_last_fetch_date}
-
-    demisto.debug(f"Fetched {len(events)} computers")
-    demisto.debug(f"{latest_event=}")
-    demisto.debug(f"{new_last_fetch_date=}")
-
-    if next_page:
-        new_last_run["next_page"] = next_page
-        demisto.debug(
-            f"Fetched the maximal number of computers. "
-            f"Fetching will continue using {next_page=} in the next iteration")
-
-    return events, new_last_run
-
-
-def get_events_audit_type(client: Client, start_date: str, end_date: str, max_fetch: int, last_run: dict) -> tuple:
-    """
-     Fetches audit type events from the Jamf Protect API within a specified date range.
-
-     This function fetches audit type events from the Jamf Protect API based on the provided start and end dates.
+     This function fetches the given type events from the Jamf Protect API based on the provided start and end dates.
      It fetches events up to the maximum number specified by max_fetch.
      The function also uses the information from the last run to continue fetching from where it left off in the previous run.
 
      Args:
-         client (Client): An instance of the Client class for interacting with the API.
          start_date (str): The start date for fetching events in '%Y-%m-%dT%H:%M:%SZ' format.
-         end_date (str): The end date for fetching events in '%Y-%m-%dT%H:%M:%SZ' format.
          max_fetch (int): The maximum number of events to fetch.
          last_run (dict): A dictionary containing information about the last run.
+         specific_type (str): The specific type to fetch.
+         client_event_type_func (Callable): The client's method which pulls the events for the given type.
+        fetch_all_computers (bool): The maximum number of events to fetch.
 
      Returns:
          tuple: A tuple containing two elements:
              - A list of dictionaries. Each dictionary represents an event.
              - A dictionary with new last run values,
               the end date of the fetched events and a continuance token if the fetched reached the max limit.
      """
-    start_date, end_date = calculate_fetch_dates(start_date, end_date=end_date, last_run=last_run, last_run_key="alert")
-    command_args = {"start_date": start_date, "end_date": end_date}
-    client_event_type_func = client.get_audits
-    next_page = last_run.get("audit", {}).get("next_page", "")
+    last_run_key = 'alert' if specific_type == 'audit' else specific_type
+    start_date, end_date = calculate_fetch_dates(start_date, last_run=last_run, last_run_key=last_run_key)
+    command_args: dict[str, Any] = {"created": start_date}
+
+    debug_message = f"Jamf Protect - Fetching {specific_type}s from {start_date} to {end_date}"
```

Comment:
while we're here, let's remove all `Jamf Protect - ` prefixes from logs. ([reason](https://panw-global.slack.com/archives/G011E63JXPB/p1730217052505469))